independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/RemoveExtensionsCommandHandler.java

```diff
@@ -44,7 +44,7 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                 final UninstallResult result = extensionManager.uninstall(keys);
                 result.getUninstalled()
                         .forEach(a -> invocation.log()
-                                .info(MessageIcons.OK_ICON + " Extension " + a.getGroupId() + ":" + a.getArtifactId()
+                                .info(MessageIcons.OK_ICON.getIcon() + " Extension " + a.getGroupId() + ":" + a.getArtifactId()
```

Comment:
Overriding toString() in the enum would avoid this change

Reply:
Done